inabox/deploy/config_types.go

```diff
@@ -99,13 +105,15 @@ func (cfg *EigenDADeployConfig) MarshalJSON() ([]byte, error) {
 	return []byte(customJSON), nil
 }
 
+// EigenDAContract is the structured output generated by running
+// forge script/Deployer.s.sol:SetupEigenDA
 type EigenDAContract struct {
 	Deployer               string `yaml:"deployer"`
 	ServiceManager         string `json:"eigenDAServiceManager"`
 	OperatorStateRetreiver string `json:"operatorStateRetriever"`
 	BlsApkRegistry         string `json:"blsApkRegistry"`
 	RegistryCoordinator    string `json:"registryCoordinator"`
-	CertVerifier           string `json:"certVerifier"`
+	CertVerifier 	   	   string `yaml:"certVerifier"`
```

Comment:
why some are yaml some are json?

Reply:
no clue tbh - the output files that are read into these structs use json representation so probably just a long standing regression